torch/_higher_order_ops/cond.py

```diff
@@ -62,7 +62,7 @@ def __call__(self, pred, true_fn, false_fn, operands):
 
 
 @exposed_in("torch")
-def cond(pred, true_fn, false_fn, operands):
+def cond(pred, true_fn, false_fn, operands=()):
```

Comment:
nit: can help add type annotations like:
```
(pred: Union[bool, torch.Tensor]. true_fn: Callable, false_fn: Callable, operands: Tuple = ())
```

Reply:
Thanks for your reveiw! ah yes I missed the doc! Will finish it now.